django/contrib/admin/sites.py

```diff
@@ -154,6 +156,13 @@ def actions(self):
         """
         return six.iteritems(self._actions)
 
+    @property
+    def empty_value_display(self):
+        return self._empty_value_display
+
+    def set_empty_value_display(self, empty_value_display):
```

Comment:
I hoped you had made this a `@empty_value_display.setter`: https://docs.python.org/3/library/functions.html#property